dietpi/dietpi-config

```diff
@@ -4540,7 +4543,7 @@ Custom         : Write = $FSBENCH_CUSTOM_WRITE | Read = $FSBENCH_CUSTOM_READ"
 
 		#Overclocking Pi1
 		# - Zero
-		echo -e "$G_HW_MODEL"
+		echo "$G_HW_MODEL"
```

Comment:
This looks like debugging left over, could be removed?

Reply:
Right? Was also wondering. And beneath, `grep -qi 'rpi zero' /DietPi/dietpi/.hw_model` should be sufficient. as `-i` ignores cases, `tr` is not needed.